README.md

```diff
@@ -9,23 +9,12 @@ is highly *unstable* just yet.
 
 ## Installation
 
-We intentionally do not provide a download for the plugin just yet. If you are
-brave enough and want to use the plugin, you have to build it from source. Note that it will download all build dependencies, so make sure you have enough space (one GB should be enough). You need Java 8 and IDEA 15. 
+You can grab the latest build [here](https://teamcity.jetbrains.com/repository/download/IntellijIdeaPlugins_Rust_Tests/.lastSuccessful/IntelliJ%20Rust-0.0.1.{build.number}.zip?guest=1).
```

Comment:
We're not yet arranged to ship builds, so i'd rather elude from providing any link just yet.